flake.nix

```diff
@@ -0,0 +1,57 @@
+{
+  description = "The Jakt Programming Language";
+  inputs = {
+    nixpkgs.url = github:NixOS/nixpkgs/nixos-unstable;
+    flake-utils.url = github:numtide/flake-utils;
+    gitignore = {
+      url = github:hercules-ci/gitignore.nix;
```

Comment:
Why do we need a gitignore inside the flake? is the one for the top level directory not sufficient?